lib/puppet.rb

```diff
@@ -36,6 +38,28 @@ module Puppet
   require 'puppet/environments'
 
   class << self
+    # TODO: Handle the rubygems case?
```

Comment:
We need to figure out how to package and ship the locale files as part of a gem. I have no idea how we'd go about doing that, TBH. I think we need to end up copying the locale files into lib/ during the gem packaging step - this is possibly something we can expand puppetlabs/packaging to handle (since it builds gems).

I don't think it's worth blocking this PR on figuring all this out, but we should file a ticket for it (probably blocking the 4.9 release) and update the TODO with the ticket numner